R/format_data.R

```diff
@@ -168,13 +170,17 @@ fmt_number <- function(data,
   # Perform input object validation
   stop_if_not_gt(data = data)
 
+  # Resolve the `locale` by choosing the default (possibly set in `gt()`) or
+  # overriding with the `locale` set here
+  locale <- resolve_locale(data = data, locale = locale)
```

Comment:
Since `validate_locale()` always seems to follow `resolve_locale()`, consider doing the validation right in `resolve_locale()`. You could add `mustWork = TRUE` as a param if you think it's possible future uses might want to opt out of validation.

Reply:
Now fixed.